solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -149,10 +149,6 @@ public void addListenerFactory(HttpListenerFactory factory) {
     this.listenerFactory.add(factory);
   }
 
-  public List<HttpListenerFactory> getListenerFactory() {
```

Comment:
Uh, you are removing a public method of an important SolrJ class?  That should not be done without backwards compatibility considerations.  Leave it please.  Mark it deprecated if you prefer.  The "add" method should be deprecated (for immutability).

Reply:
Ah, very good then